### DISCUSSION TITLE
pengwin-setup in Pengwin Enterprise

### DISCUSSION THREAD

**Discussion by ghost** [WhitewaterFoundry/Pengwin#674] [Q&A]
hi is there any way to use pengwin-setup in the enterprise version need a quick way to set up my distro

**Comment by crramirez** [comment]
Hello,

Currently you cannot run pengwin-setup in Pengwin Enterprise. The good new is that we are working on that.

What do you need to set up and I will paste the commands here.

Regards,
Carlos

**Comment by ghost** [comment]
Hello dear, Carlos I somehow managed to run the setup by downloading the pengwin-setup-master repo from GitHub and install apt and other needed packs. It is good to hear you are working on the enterprise version too and also I appreciate your help. Have a good time regards. Sent from Mail for Windows 10

**Reply by crramirez** [reply]
Ok, I am very impressed that this is working. Excellent
